src/time/format/custom/http_date.cr

```diff
@@ -61,7 +61,7 @@ struct Time::Format
         char '-'
         short_month_name
         char '-'
-        year_modulo_100
+        full_or_short_year
```

Comment:
This derivates from the described format. The change is fine, but could you please mention the non-RFC-compliant IIS date format in the docs and add a code comment here to reference that?

Reply:
Done.